mech_client/cli.py

```diff
@@ -58,10 +69,131 @@
 from scripts.nvm_subscribe import main as nvm_subscribe_main
 
 
+CURR_DIR = Path(__file__).resolve().parent
+BASE_DIR = CURR_DIR.parent
+GNOSIS_TEMPLATE_CONFIG_PATH = BASE_DIR / "config" / "mech_client.json"
+OPERATE_FOLDER_NAME = ".operate_mech_client"
+OPERATE_CONFIG_PATH = "services/sc-*/config.json"
+OPERATE_KEYS_DIR = "services/sc-*/deployment/agent_keys"
+DEFAULT_NETWORK = "gnosis"
+
+
+def get_operate_path() -> Path:
+    """Fetches the operate path for the mech client service"""
+    home = Path.home()
+    operate_path = home.joinpath(OPERATE_FOLDER_NAME)
+    return operate_path
+
+
+def my_configure_local_config(
+    template: ServiceTemplate, operate: "OperateApp"
+) -> QuickstartConfig:
+    """Configure local quickstart configuration."""
+    config = load_local_config(operate=operate, service_name=template["name"])
+
+    if config.rpc is None:
+        config.rpc = {}
+
+    for chain in template["configurations"]:
+        config.rpc[chain] = os.getenv("MECHX_RPC_URL")
+
+    config.principal_chain = template["home_chain"]
+
+    # set chain configs in the service template
+    for chain in template["configurations"]:
+        template["configurations"][chain] |= {
+            "staking_program_id": NO_STAKING_PROGRAM_ID,
+            "rpc": config.rpc[chain],
+            "cost_of_bond": 1,
+        }
+
+    if config.user_provided_args is None:
+        config.user_provided_args = {}
+
+    config.store()
+    return config
+
+
+def fetch_agent_mode_data(chain_config: Optional[str]) -> Tuple[str, str]:
```

Comment:
Why not use service_manager and keys_manager here instead of managing paths, etc. by hand ?